README.md

```diff
@@ -1,25 +1,35 @@
-[![Build status](https://travis-ci.org/rust-embedded/r0.svg?branch=master)](https://travis-ci.org/rust-embedded/r0)
+# r0
+
+[![Build status](https://api.travis-ci.org/rust-embedded/r0.svg?branch=master)](https://travis-ci.org/rust-embedded/r0)
 [![crates.io](https://img.shields.io/crates/d/r0.svg)](https://crates.io/crates/r0)
 [![crates.io](https://img.shields.io/crates/v/r0.svg)](https://crates.io/crates/r0)
 
-# `r0`
+Memory initialization code **crt0** written in Rust. This crate is for bare metal systems where there is no ELF loader or OS to take care of initializing RAM for the program. Is also provides an API to add «life before main» functionality to bare metal systems.
+
+This project is used by other crates, like:
+
+* [cortex-m-rt](https://github.com/rust-embedded/cortex-m-rt)
+* [riscv-rt](https://github.com/rust-embedded/riscv-rt)
+* [msp430-rt](https://github.com/rust-embedded/msp430-rt)
 
-Memory initialization code ("[crt0]") written in Rust.
+Wiki quote:
+
+> crt0 (also known as c0) is a set of execution startup routines linked into a C program that performs any initialization work required before calling the program's main function.
+> It generally takes the form of an object file called crt0.o, often written in assembly language, which is automatically included by the linker into every executable file it builds.
+
+Read more about [crt0](https://en.wikipedia.org/wiki/Crt0).
 
 This project is developed and maintained by the [Cortex-A, Cortex-M, Cortex-R, MSP430, and RISCV
 teams][teams].
 
-[crt0]: https://en.wikipedia.org/wiki/Crt0
-
 ## [Documentation](https://docs.rs/r0)
 
 ## License
 
 Licensed under either of
 
-- Apache License, Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE) or
-  http://www.apache.org/licenses/LICENSE-2.0)
-- MIT license ([LICENSE-MIT](LICENSE-MIT) or http://opensource.org/licenses/MIT)
+* Apache License, Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE) or [http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0))
+* MIT license ([LICENSE-MIT](LICENSE-MIT) or [http://opensource.org/licenses/MIT](http://opensource.org/licenses/MIT))
```

Comment:
These changes are not necessary

Reply:
Yes, but if you use any linter of markdown (like VSS Code plugin), it will say, that raw links are prohibited.

Reply:
You can use `<link>` syntax here to reduce duplication

Reply:
While you're changing these, using `https` instead of `http` is probably a good idea too